scripts/update_robot.py

```diff
@@ -183,7 +184,55 @@ def on_shutdown():
     return nl.rc
 
 
+def rethink_ssh_update_version():
+    """
+    Verifies the version of the software is older than 1.1.0.
+    If newer, print out url to SSH update instructions
+    and return False
+    """
+    param_name = "rethink/software_version"
+    robot_version = rospy.get_param(param_name, None)
+    if not robot_version:
+        rospy.logerr("RobotUpdater: Failed to retrieve robot version "
+                     "from rosparam: %s\n"
+                     "Verify robot state and connectivity "
+                     "(i.e. ROS_MASTER_URI)", param_name)
+        return False
+    else:
+        # parse out first 3 digits of robot version tag
+        pattern = ("^([0-9]+)\.([0-9]+)\.([0-9]+)")
+        match = re.search(pattern, robot_version)
+        if not match:
+	    rospy.logwarn("RobotEnable: Invalid robot version: %s",
```

Comment:
I suppose this isn't the time (given schedule constraints), but this addition might be the motivation for next next release to move the version checking code into a baxter_interface `version` module (we talked about doing this before, where we could also add fuller logic code for dealing with version numbers on robot, workspace, and grippers.

Then again, I haven't heard about versioning problems in a while, so maybe this isn't a 'need'.